### DISCUSSION TITLE
pylogix used in Micropython code

### DISCUSSION THREAD

**Discussion by akashkumarmandal** [dmroeder/pylogix#189] [General]
Can we use pylogix library in micropython...if not suggest compatible library to use for reading tags from PLC

**Comment by TheFern2** [comment]
Hey there we had one issue with the same question https://github.com/dmroeder/pylogix/issues/145 that user never reported back if it works so we can't say for sure if it works, from what I read that day all the python modules used on pylogix are in micropython, but are prefixed with an u. So you will need to change that in the pylogix source code.

**Comment by TheFern2** [comment]
Micropython is now supported check out documentation. Did you ever find an alternative solution?

https://github.com/dmroeder/pylogix/issues/145#issuecomment-1767288641
